docs/deployment/deploy_gardenlet_automatically.md

```diff
@@ -0,0 +1,28 @@
+# Automatic Deployment of Gardenlets
+
+The gardenlet can automatically deploy itself into shoot clusters and register this cluster as a seed cluster. You refer to these clusters as shooted seeds. This procedure is the preferred way to add additional seed clusters, because shoot clusters already come with production-grade qualities that are also demanded for seed clusters.
+
+## Prerequisites
+
+The only prerequisite is to register an initial cluster as a seed cluster that has already a gardenlet deployed:
+
+* This gardenlet was either deployed as part of a Gardener installation or

```

Comment:
@danielfoehrKn : Since the Gardener team works on a new installation procedure, how stable would this link be? ;-)

My main issue here is that I am not sure if I can move this link to the NS2 documentation (or the CDC documentation). If you think the link is important, could I move it to the _Related Links_ section? It would make it easier for me to remove links that are not relevant for the private cloud shipments, before I import it to our DITA CMS.

Reply:
Sure Related Links makes sense.  Would still like to add that you mean an Gardener landscape that was setup by an installer tool (as a landscape can setup in many ways)

Reply:
I added a related link to `gardener/garden-setup` in the related links section, because `gardener/landscape-setup` is deprecated.

In the text for the bullet, I didn't provide a link, but just refer textually to `garden-setup`, hope that's o.k.